tfjs-core/src/kernel_names.ts

```diff
@@ -813,6 +813,15 @@ export interface FromPixelsAttrs {
   numChannels: number;
 }
 
+export const FromPixelsAsync = 'FromPixelsAsync';
+export interface FromPixelsAsyncInputs {
```

Comment:
It seems that `FromPixelsAsyncInputs` and `FromPixelsAsyncAttrs` are not used anywhere. Please remove them if they are not needed.